### PR TITLE
added `new state` command (#21)

### PR DESCRIPTION
**Pull request by baljinderxd**
for issue #21

### CODE REVIEW

**Comment by gurrrung** [comment]
Hey, thanks for the PR. I just integrated lint checks with Github Action in this repo. Thus I have refactored a big chunk of this project following the lint errors. Can you remove the conflicts and make the check pass?

**Comment by gurrrung** [comment]
Further, make sure `npm run lint` doesn't give you an error locally.

**Comment by baljinderxd** [comment]
> Further, make sure `npm run lint` doesn't give you an error locally.

there are some errors, i will correct them and then will commit the updated code.

**Comment by baljinderxd** [comment]
please check if there is any mistake, this lint thing was new for me . And yes `npm run lint` is not giving any error now.

File: README.md [hunk 1]
```diff
@@ -12,7 +12,11 @@
 | daily N                 | Gives all india daily changes analysis for past `N` days    | :heavy_check_mark: | `daily 15`     |
 | test State              | Gives all the test centers for Corona in the given state    | :heavy_check_mark: | `test punjab`  |
 | new                     | Gives all the states with sorted new cases                  | :heavy_check_mark: | `new`          |
+<<<<<<< HEAD
```
Comment (gurrrung) [inline README.md:15]: You can remove these unwanted arrows and HEAD.

Comment (baljinderxd) [reply README.md:15]: Okay will do it, didn't knew they were there 😬

**Comment by gurrrung** [comment]
Thanks, for the PR. Great, work.

**Pull request merged**